sdk/typescript/src/providers/provider.ts

```diff
@@ -362,4 +363,11 @@ export abstract class Provider {
    */
   abstract getReferenceGasPrice(): Promise<number>;
   // TODO: add more interface methods
+
+  /**
+   * Return the committee information for the asked epoch
+   * @param epoch The epoch of interest. If None, default to the latest epoch
```

Comment:
```suggestion
   * @param epoch The epoch of interest. If null, default to the latest epoch
```
nit: Do you mind also moving line 365 to the bottom?

Reply:
Yes of course, I also added the e2e tests 🙂